ml/db/init.sql

```diff
@@ -70,45 +79,54 @@ INSERT INTO gamemode (num, name) VALUES
 	(4,'Snowday'),(5,'RocketLabs'),(6,'Tournament'),(7,'DropshotRumble'),
 	(8,'Heatseeker'),(9,'Gridiron'),(10,'Private'),(11,'Season'),
 	(12,'Offline'),(13,'LocalLobby');
+
+-- Make it so num smallserial starts at 14, the next highest num value possible
+ALTER SEQUENCE gamemode_num_seq RESTART WITH 14;
 	
 -- GAMETYPES
 -- Gametype relation
 CREATE TABLE gametype (
-	num					smallint		CHECK (num >= 0 AND num <= 4),
+	num					smallserial,
 	name				text,
 	PRIMARY KEY (num));
 	
 -- Required gametype data
 INSERT INTO gametype (num, name) VALUES
 	(0,'Duels'),(1,'Doubles'),(2,'Standard'),(3,'SoloStandard'),(4,'Chaos');
+
+-- Make it so num smallserial starts at 5, the next highest num value possible
+ALTER SEQUENCE gametype_num_seq RESTART WITH 5;
```

Comment:
Good thinking with the smart serial, had no idea you can alter the sequence like that suck. Great!